src/mame/drivers/plygonet.cpp

```diff
@@ -74,15 +74,35 @@
 #include "speaker.h"
 
 
+#define LOG_DSP_AB0			(1U << 1)
+#define LOG_DSP_A6			(1U << 2)
+#define LOG_DSP_A7			(1U << 3)
+#define LOG_DSP_A8			(1U << 4)
+#define LOG_DSP_AC			(1U << 5)
+#define LOG_DSP_AE			(1U << 6)
+#define LOG_DSP_B6			(1U << 7)
+#define LOG_DSP_B7			(1U << 8)
+#define LOG_DSP_B8			(1U << 9)
+#define LOG_68K_SHARED		(1U << 10)
+#define LOG_DSP_HOST_INTF	(1U << 11)
+#define LOG_DSP_CTRL        (1U << 12)
+#define LOG_DSP_PORTC       (1U << 13)
+
+#define LOG_ALL_DSP_A       (LOG_DSP_AB0 | LOG_DSP_A7 | LOG_DSP_A6 | LOG_DSP_A8 | LOG_DSP_AC | LOG_DSP_AE)
+#define LOG_ALL_DSP_B		(LOG_DSP_AB0 | LOG_DSP_B6 | LOG_DSP_B7 | LOG_DSP_B8)
```

Comment:
Can you please srcclean this file?  It will save a bit of churn later.

Reply:
Done.